bundle/direct/dresources/adapter.go

```diff
@@ -306,6 +314,13 @@ func (a *Adapter) validate() error {
 		}
 	}
 
+	if a.classifyChange != nil {
+		validations = append(validations, "ClassifyChange changes", a.classifyChange.InTypes[1], stateType)
+		if len(a.classifyChange.OutTypes) == 3 {
+			validations = append(validations, "ClassifyChange remoteState return", a.classifyChange.OutTypes[1], remoteType)
+		}
+	}
+
```

Comment:
We should either

- validate that if ClassifyChange is present, FieldTriggers is not and vice versa.
OR
- we can allow both to be specified but then we use both. ClassifyChange is called first but if it returns ActionTypeUnset, we fallback to FieldTriggers. 

The second one seems most convenient, might even be useful for clusters to recreate/skip on some fields.

Reply:
Done here https://github.com/databricks/cli/pull/3690/commits/6212f0d287ff5d909af1e955b9bb73297b1cb996